public/js/bugzilla.js

```diff
@@ -31,12 +31,13 @@ function bugzilla(colors, product){
       element: 'bug_prio',
       data: json,
       xkey: 'bugprio',
-      ykeys: ['p1', 'p2', 'p3', 'p4', 'p5'],
-      labels: ['P1 - Urgent', 'P2 - High', 'P3 - Medium', 'P4 - Low', 'P5 - None'],
+      ykeys: ['p0', 'p1', 'p2', 'p3', 'p4', 'p5'],
+      labels: ['P0 - Crit Sit', 'P1 - Urgent', 'P2 - High', 'P3 - Medium', 'P4 - Low', 'P5 - None'],
       resize: true,
       stacked: true,
       hideHover: true,
-      barColors: [ colors["bar"]["red"],
+      barColors: [ "#000000",
```

Comment:
please add the color to the `colors.yml`, sth like this

``` yaml
bar:
  black: "#000000"
  red: "#E57373"
```

and then call it like `colors["bar"]["black"]`